### ISSUE TITLE
Allow customization of CalendarList loading placeholder

### ISSUE THREAD

**Issue by jeferson-sb** [wix/react-native-calendars#2459]
## Description

Whenever you have CalendarList loading there is a quick flash of a `<Text>` showing for each of the months (calendars) being rendered. But looking into the props there were no options to either replace the text or change its string.

I found out to be this guy: 

https://github.com/wix/react-native-calendars/blob/da9b1bd702e1221002d6e30aaf888e350b627b6a/src/calendar-list/item.tsx#L84-L88  

And the styles:

https://github.com/wix/react-native-calendars/blob/da9b1bd702e1221002d6e30aaf888e350b627b6a/src/calendar-list/style.ts#L14-L23  

## Expected Behavior

I expect to be able to at least change the style of the placeholder text.

## Observed Behavior

See example below:

**Pay attention to the 2024-05-01 text showing in the place of the calendar**
 
https://github.com/wix/react-native-calendars/assets/30840709/f1139327-3a0a-4221-8f14-60ddc892b5c7

Code: 
```tsx
import React from 'react';
import {CalendarList} from 'react-native-calendars';

export const MyCalendar = () => {
  return (
    <CalendarList
      showScrollIndicator
      pastScrollRange={5}
      futureScrollRange={5}
    />
  );
};

```

**Comment by sifthedog** [comment]
I would also like this to be implemented. I can submit a PR

**Comment by jeferson-sb** [comment]
I was able to circumvent this for now changing the theme with:

```ts
const calendarTheme = {
  'stylesheet.calendar-list.main': {
    placeholder: {
       // wrapper styles
    },
    placeholderText: {
      // text styles
    },
  },
};

export default calendarTheme;
```

**Comment by vagnereix** [comment]
> Consegui contornar isso por enquanto mudando o tema com:

You change the text as well or just the styles? @jeferson-sb

**Comment by jeferson-sb** [comment]
@vagnereix Only styles

**Comment by lestercorderomurillo** [comment]
yes, i hate this, how this lib cannot have something as basic as a loader component customizable? terrible

**Comment by sifthedog** [comment]
@lestercorderomurillo you are always welcome to submit a PR for this or fork the library and implement it yourself.

Kindly remember that you are not paying for this library and people developing it are not getting any money from it. 

I would recommend you being more respectful towards the people that helps building open source software. As, without them, you wouldn't even have a place to call another's person work terrible 😄

**Comment by Sunhat** [comment]
My team need this too. I'll submit the PR.

**Comment by Sunhat** [comment]
Goodness. Totally forgot to do this! I'll work on it now.

**Comment by gdeak-monguz** [comment]
> Goodness. Totally forgot to do this! I'll work on it now.

Thanks for completing this! Do you know in which release this will be included?
